internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,43 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	unpacked := strings.Split(data, ",")
+	if len(unpacked) != 2 {
+		return 0, 0, fmt.Errorf("unproper string")
+	}
+	steps, err := strconv.Atoi(unpacked[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать все ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```